robot/Cumulus/resources/NPSP.py

```diff
@@ -511,6 +511,7 @@ def verify_occurence(self,title,value):
            
         
     def check_record_related_item(self,title,value):
+        self.salesforce.load_related_list(title)
```

Comment:
can you add documentation to this keyword?

Reply:
ahh missed it, will add